### ISSUE TITLE
MRR broken connector

### ISSUE THREAD

**Issue by philrosenberg** [philrosenberg/instrument-log#1]
The data connector on the dish arm was broken. The plastic insert was moving within the shell. This meant that when connecting the cable the plastic insert pushed into the shell rather than engaging the pins properly.

During the Reunion project Alastair opened up the box and was able to put some pressure on the back face of the connector to get it to engage properly.

**Comment by philrosenberg** [comment]
Replaced the broken connector. In the end, I only needed to replace the shell. I found a bit of plastic rattling around in the electronics box that I think was the ring that should have been holding the plastic insert in the shell. The old insert just slid out of the shell and slid into the new shell and clicked into place.

The connector was available from RS. The manufacturer is Phoenix Contact and it is RS part number 822-5175 and manufacturer part number 1619972. I've attached the datasheet - note that this is a 9 pin connector even though the image on the data sheet is 7 pin.

The screws holding the connector to the box were siliconed on the inside - I guess for watertightness. One of the screws had an earthing tag on the back and the screwhead for this screw sheared off - worth noting if whoever reads this needs to do something similar. I replaced all the connector screws - mostly because we had them, so I thought I might as well so they were all the same. I didn't put new silicone on, but I did check continuity on the earth tag. Might be worth adding some non-permenant threadlock to the screws to ensure they are watertight. I'll raise another issue for that.


[0900766b81338b97.pdf](https://github.com/philrosenberg/instrument-log/files/6325745/0900766b81338b97.pdf)
